.github/workflows/fuzzing.yml

```
@@ -133,12 +151,14 @@
         mkdir -p artifacts
         chmod a+x ubpf_fuzzer
 
+
     - name: Run fuzzing
+      if: matrix.platform == 'ubuntu-24.04'
       run: |
         ./ubpf_fuzzer new_corpus -artifact_prefix=artifacts/ -use_value_profile=1 -max_total_time=300
 
     - name: Merge corpus into fuzz/corpus
-      if: ${{ github.event_name == 'schedule' }}
+      if: ${{ github.event_name == 'schedule' }} && matrix.platform == 'ubuntu-24.04'
```

actionlint

if: condition "${{ github.event_name == 'schedule' }} && matrix.platform == 'ubuntu-24.04'" is always evaluated to true because extra characters are around ${{ }}

(if-cond)